.gitignore

```diff
@@ -2,3 +2,4 @@
 .DS_Store
 dist
 node_modules
+.idea
```

Comment:
Thanks @kfitzgerald! I was going to merge this PR, but I'm not sure what this line is for. Do you mind removing it?